src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -781,6 +781,17 @@ void SolverInterfaceImpl::setMeshQuad(
   }
 }
 
+// First organise four points into 2 sets of 3 vertices. 
+// Create two triangles with the four points
+/*
+  2 ------- 3
+    |\    |
+    | \   |
+    |  \  |
+    |   \ |
+    |    \|
+  1 ------- 0
+*/
```

Comment:
That's an unusual place for a comment. Either place it before the definition in the header file or inside the body of the function.